clients/lsp-eslint.el

```diff
@@ -308,9 +351,9 @@ to allow or deny it.")
   :activation-fn (lambda (filename &optional _)
```

Comment:
not part of this review, but at some point we might use `lsp-activate-on` here, which will simplify the code a lot.